debian/control

```diff
@@ -0,0 +1,20 @@
+Source: golang-logrus
+Section: devel
+Priority: extra
+Maintainer: Tianon Gravi <admwiggin@gmail.com>
+Uploaders: Docker Packaging Team <docker-maint@lists.alioth.debian.org>
```

Comment:
To explain a little here, `Maintainer` is the main go-to lead on the package.  `Uploaders` is basically "the other maintainers", so you're not alone here and there's nothing scary to it! :+1:

If you're still frightened though, you should at least add yourself to `Uploaders`.  I'm OK being in the `Maintainer` slot, but if you want it, it's yours.  You've obviously done all the work here. :+1::+1::+1: